### PR TITLE
[ISSUE #5678] Add logging exporter for metrics

### PR DESCRIPTION
**Pull request by lrybbs**
I have completed the issue #5678 @ShadowySpirits , now broker and proxy could export metrics in terminal.

### CODE REVIEW

File: common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java [hunk 1]
```diff
@@ -376,12 +379,13 @@ public boolean isEnable() {
         }
     }
 
-    private MetricsExporterType metricsExporterType = MetricsExporterType.DISABLE;
+    private MetricsExporterType metricsExporterType = MetricsExporterType.LOGGER;
```
Comment (ShadowySpirits) [inline common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java:382]: Please do not change the default value.

**Review by ShadowySpirits — CHANGES_REQUESTED** [review]

File: common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java (hunk 1 above)
Comment (ShadowySpirits) [reply common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java:382]: @aaron-ai Could you please approve running the ci? And we need to add a jul to sl4j bridge in [aliyunmq/rocketmq-shaded-slf4j-api-bridge](https://github.com/aliyunmq/rocketmq-shaded-slf4j-api-bridge/tree/master).

Comment (aaron-ai) [reply common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java]: Actually there is [an official bridge from SLF4j](https://www.slf4j.org/legacy.html#:~:text=endless%20loop.-,jul%2Dto%2Dslf4j%20bridge,-The%20jul%2Dto).

Comment (ShadowySpirits) [reply common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java]: > Actually there is [an official bridge from SLF4j](https://www.slf4j.org/legacy.html#:~:text=endless%20loop.-,jul%2Dto%2Dslf4j%20bridge,-The%20jul%2Dto), @lrybbs just try it out.

You are right, but we should make it compatible with shaded sl4j. Can you give some guide for @lrybbs?

Comment (lrybbs) [reply common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java]: @aaron-ai Now, broker metrics and proxy metric could output to "rocketmqlogs" folder. Their names are "broker_metric.log" and "proxy_metric.log", respectively.

**Review by drpmma — APPROVED** [review]
LGTM

**Pull request merged**